src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -25,6 +25,8 @@ def sdkVersion = request.getProperties().get("sdkVersion")
 def includeDispatcherConfig = request.getProperties().get("includeDispatcherConfig")
 def includeCommerce = request.getProperties().get("includeCommerce")
 def includeForms = request.getProperties().get("includeForms")
```

Comment:
Why can't we just use the same definition includeForms here ?
```
def includeForms = "n"
        request.getProperties().entries().each { item ->
            if (item.getKey().equalsIgnoreCase("includeforms") && item.getValue().equalsIgnoreCase("y")) {
                includeForms = "y"
            } ....
        }
```

Reply:
That is possible but we'd need to change it again when `includeForms` is removed in the near future. The current version is consistent with changes elsewhere as well. I can make this change if required though.